api/controllers/userController.js

```diff
@@ -0,0 +1,39 @@
+const {userService} = require("../services");
+const {catchAsync} = require("../utils/error");
+
+const oauth = catchAsync(async (req, res) => {
```

Comment:
```suggestion
const kakaoSignIn = catchAsync(async (req, res) => {
```
OAuth는 인터넷 사용자들이 직접 회원가입과 같은 것을 통해 비밀번호를 제공하지 않고 다른 웹사이트 상의 자신들의 정보에 대해 웹사이트나 애플리케이션의 접근 권한을 부여할 수 있는 공통적인 수단으로서 사용되는 프로토콜입니다. OAuth같은 기술적인 개념의 용어로 정의하기 보다 실제로 해당 함수가 뭘 위한건지를 명시해주는것이 좋겠습니다.